src/hosts/gitlab.ts

```diff
@@ -18,6 +18,17 @@ export function injectionScope(url: string) {
 			this.insertHTML(insertions);
 		}
 
+	  /*
+			** PLEASE READ BEFORE TOUCHING ANY INJECTIONS **
+
+			GitLab supports up to two major versions behind the current version. See:
+			https://about.gitlab.com/support/statement-of-support/#version-support
+
+			In addition, GitLab supports the previous two monthly releases of the current stable release:
+			https://docs.gitlab.com/ee/policy/maintenance.html
+
+			Please keep this in mind, as this code is also used for Self-Managed!
+		*/
```

Comment:
Technically this isn't relevant here, but I'm going to be adding self-hosted support this week.